src/collections/blog.item

```diff
@@ -2,93 +2,130 @@
 {.section item}
 
   <!--WRAPPER-->
-  
-  <article id="post-{id}" class="{@|item-classes}" data-item-id="{id}">
-    
-    <!--POST TILE-->
-
-    <h1 class="title" data-content-field="title">
-      {.passthrough?}
-        <a href="{sourceUrl}" target="_blank">{title}</a>
-      {.or}
-        <a href="{fullUrl}">{title}</a>
-      {.end}
-    </h1>
-
-    <!--MAIN CONTENT-->
-
-    {body}
-
-    <!--BLOG INJECTION-->
-
-    {postItemInjectCode}    
-
-    <!--CATEGORIES-->
-
-    {.repeated section categories}
-      <a class="category" href="{collection.fullUrl}?category={@|url-encode}">{@}</a>{.alternates with},
-    {.end}
-
-    <!--TAGS-->
-
-    {.repeated section tags}
-      <a class="tag" href="{collection.fullUrl}?tag={@|url-encode}">{@}</a>{.alternates with},
-    {.end}
-
-    <!--SHARE AND LIKE-->
-
-    {@|like-button}
-    {@|social-button}
 
-    <!--AUTHOR AND DATE-->
-    
-    <p class="meta">
-      Published on <a href="{fullUrl}"><time datetime="{addedOn|date %F}">{addedOn|date %B %d, %Y}</time></a> by <a href="{collection.fullUrl}?author={author.id}">{author.displayName}</a>.
-      {.section sourceUrl}
-        <a href="{@}" class="source" target="_blank">Source</a>.
-      {.end}
-    </p>        
-
-    <!--LOCATION-->
-
-    {.section location}
-      {.section addressTitle}
-        {addressTitle}
-        {addressLine1}
-        {addressLine2}
-        {addressCountry}
-      {.end}
-    {.end}
-
-    <!--COMMENTS-->
-
-    {@|comments}
-
-  </article>
+  <div class="row">
```

Comment:
Let's make sure we're using 4 spaces across all of our markup and templates in this project